niworkflows/anat/ants.py

```diff
@@ -486,14 +413,53 @@ def init_brain_extraction_wf(
                 ('outputnode.out_segm', 'out_segm'),
                 ('outputnode.out_tpms', 'out_tpms'),
             ]),
-        ])
-        # fmt: on
+        ])  # fmt:skip
         if wm_tpm:
-            # fmt: off
-            wf.connect([
-                (map_wmmask, atropos_wf, [('output_image', 'inputnode.wm_prior')]),
-            ])
-            # fmt: on
+            wf.connect([(map_wmmask, atropos_wf, [('output_image', 'inputnode.wm_prior')])])
+    else:
+        # If no Atropos refinement, rerun N4 with the brain or white matter mask
+        inu_n4_final = pe.MapNode(
+            N4BiasFieldCorrection(
+                dimension=3,
+                save_bias=True,
+                copy_header=True,
+                n_iterations=[50] * 5,
+                convergence_threshold=1e-7,
+                shrink_factor=4,
+                bspline_fitting_distance=bspline_fitting_distance,
+            ),
+            n_procs=omp_nthreads,
+            name='inu_n4_final',
+            iterfield=['input_image'],
+        )
+
+        # Check ANTs version
+        try:
+            inu_n4_final.inputs.rescale_intensities = True
+        except ValueError:
+            warn(
+                "N4BiasFieldCorrection's --rescale-intensities option was added in ANTS 2.1.0 "
+                f'({inu_n4.interface.version} found.) Please consider upgrading.',
```

Comment:
not that it really matters but
```suggestion
                f'({inu_n4_final.interface.version} found.) Please consider upgrading.',
```